scripts/spack/packages/axom/package.py

```diff
@@ -91,6 +92,10 @@ class Axom(CachedCMakePackage, CudaPackage):
     # -----------------------------------------------------------------------
     # Basics
     depends_on("cmake@3.8.2:", type='build')
+    depends_on("cmake@3.8.2:3.14.5", type='build', when="+cuda")
```

Comment:
This is not true.. we can definitely build higher than 3.14.5 when building with cuda.